client/src/lib/dependency-injection/inject.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import injectorContext from './injectorContext';
 
-const inject = (Component, dependencies, mapDependenciesToProps) => {
+const inject = (dependencies, mapDependenciesToProps, contextArg) => (Component) => {
   // eslint-disable-next-line react/prefer-stateless-function
   class Injected extends React.Component {
     render() {
       let props = {};
+      const context = typeof contextArg === 'function' ?
```

Comment:
Yeah, good catch. Enforced types for both args. `dependencies` was an array everywhere it was being used anyway.